filebeat/docs/getting-started.asciidoc

```diff
@@ -99,6 +106,27 @@ To see the full list of variables for a module, see the documentation under
 
 include::{libbeat-dir}/shared/config-check.asciidoc[]
 
+[float]
+[[collect-application-logs]]
+==== Enable and configure ECS loggers for application log collection
+
+While {beatname_uc} can be used to ingest raw, plain-text application logs,
+we recommend structuring your logs at ingest time. This lets you extract fields,
+like log level and exception stack traces.
+
+Elastic simplifies this process by providing application log formatters in a variety
+of popular programming languages. These plugins format your logs into ECS compatible JSON,
+which removes the need to manually parse logs.
+
+See {ecs-logging-ref}/intro.html[ECS loggers] to get started.
+
+[float]
+[[manual-configuration]]
+==== Manual configuration
```

Comment:
```suggestion
==== Configure {beatname_uc} manually
```

Reply:
for parallelism. I'm on a kick, it seems. :-P

Reply:
These are great! Thank you.